### PR TITLE
feat(query): added 'Object Without Required Property' for swagger

### PR DESCRIPTION
**Pull request by rafaela-soares**
Signed-off-by: Rafaela Soares <rafaela.soares@checkmarx.com>

**Proposed Changes**
- Added 'Object Without Required Property' for openAPI 2.0

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Review by felipe-avelar — DISMISSED** [review]
Please, check comments

File: assets/queries/openAPI/2.0/object_without_required_property/query.rego [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	object.get(param, "items", "undefined") == "undefined"
+
+	result := {
+		"documentId": doc.id,
+		"searchKey": sprintf("%s.{{%s}}", [partialSk, n]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "Parameter object has 'items' defined",
+		"keyActualValue": "Parameter object does not have 'items' defined",
+	}
+}
```
[the remark this answers is not in the document]
Comment (rafaela-soares) [reply assets/queries/openAPI/2.0/object_without_required_property/query.rego:66]: It is not the same. Here, we want `type == array`

[the remark this answers is not in the document]
Comment (felipe-avelar) [reply assets/queries/openAPI/2.0/object_without_required_property/query.rego:66]: Sorry, you're right, I was trying tu link [these query](https://github.com/Checkmarx/kics/blob/master/assets/queries/openAPI/general/schema_items_undefined/query.rego)

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Pull request merged**